staging/src/k8s.io/apimachinery/pkg/runtime/converter.go

```diff
@@ -438,13 +437,15 @@ func (c *unstructuredConverter) ToUnstructured(obj interface{}) (map[string]inte
 }
 
 // DeepCopyJSON deep copies the passed value, assuming it is a valid JSON representation i.e. only contains
-// types produced by json.Unmarshal().
+// types produced by json.Unmarshal() and also int64.
+// bool, int64, float64, string, []interface{}, map[string]interface{}, json.Number and nil
```

Comment:
is this an enumeration of "types produced by json.Unmarshal() and also int64" ?

nit: put a `:` behind the previous sentence.